assets/python/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Find all child directories with a requirements.txt file
+directories=(*/)
+
+# Loop through the child directories and run run.sh if it exists
+for dir in "${directories[@]}"; do
+    if [ -f "${dir}run.sh" ]; then
+        echo "Running ${dir}run.sh..."
+        cd "${dir}"
+        bash "run.sh"
+        echo "Execution of ${dir}run.sh completed."
+        cd ..
+    fi
+done
+
+ls -la
```

Comment:
These can be removed and where only for verbosity

Reply:
yes pls 👉👈